analysis/escape/escape_test.go

```diff
@@ -97,7 +97,7 @@ func TestSimpleEscape(t *testing.T) {
 	t.Parallel()
 
 	dirName := filepath.Join("./testdata", "simple-escape")
-	lp, err := analysistest.LoadTest(testfsys, dirName, []string{})
+	lp, err := analysistest.LoadTest(testfsys, dirName, []string{}, true)
```

Comment:
nit: Can we make `LoadTest` take an options struct as well? It's difficult to know what `true` refers to when reading